Donation-platform/src/main/webapp/frontend/src/getTagScores.js

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+///home/anerishah/step-donation/step28-2020/Donation-platform/src/main/webapp/frontend/src
+///usr/local/nvm/versions/node/v10.14.2/bin/node
+
+var googleTrends = require('/home/anerishah/step-donation/step28-2020/Donation-platform/src/main/webapp/frontend/node_modules/google-trends-api/lib/google-trends-api.min.js');
```

Comment:
Won't using this path cause issues if it is not a relative path? I believe Ryan was having this issue in his branch.